v3/tasks/20250214-v3-protocol-fee-controller-v2/readme.md

```diff
@@ -0,0 +1,11 @@
+# 2025-02-14 - V3 Protocol Fee Controller (V2)
+
+Redeployment of the Protocol Fee Controller. This version adds events on new pool registration, reporting the initial values of the aggregate swap and yield fees. Otherwise, it would be much harder to track protocol fee percentages for new pools (and actually impossible to know whether they were protocol fee exempt).
```

Comment:
```suggestion
Redeployment of the Protocol Fee Controller. This version adds events on new pool registration, reporting the initial values of the aggregate swap and yield fees. Otherwise, it would be much harder to track protocol fee percentages for new pools (and actually impossible to know whether they were protocol fee exempt solely by tracking events).
```